File: apps/ios/PagApp/PagApp/Models/UserProfileMock.swift
import Foundation

public struct UserProfileMock: Identifiable {
    public let id: String = UUID().uuidString
    public let name: String
    public let profileScore: Int
    public let rankingAdvantageText: String
    public let rankingPercentileText: String
    
    public static let sample = UserProfileMock(
        name: "Metin",
        profileScore: 12480,
        rankingAdvantageText: "Öncelikli Sıra",
        rankingPercentileText: "İlk %8"
    )
}

File: apps/android/app/src/main/java/com/alafteknoloji/pagapp/models/UserProfileMock.kt
package com.alafteknoloji.pagapp.models

import java.util.UUID

data class UserProfileMock(
    val id: String = UUID.randomUUID().toString(),
    val name: String,
    val profileScore: Int,
    val rankingAdvantageText: String,
    val rankingPercentileText: String
) {
    companion object {
        val sample = UserProfileMock(
            name = "Metin",
            profileScore = 12480,
            rankingAdvantageText = "Öncelikli Sıra",
            rankingPercentileText = "İlk %8"
        )
    }
}
